src/ray/raylet/io_spill_worker_pool.h

```diff
@@ -0,0 +1,52 @@
+// Copyright 2017 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include "ray/object_manager/plasma/spill_worker_pool.h"
+#include "ray/raylet/worker_pool.h"
+
+namespace ray {
+namespace raylet {
+class IOSpillWorkerPool : plasma::ISpillWorkerPool {
+ public:
+  IOSpillWorkerPool(instrumented_io_context &worker_pooler_executor,
+                    IOWorkerPoolInterface &io_worker_pool);
+
+  void StartSpillWorker(
```

Comment:
Pretty much same comment as previous PR. We only seem to have Start*, but not like Dispatch* APIs. Is `StartSpillWorker ` if we need to start workers, and otherwise it will only dispatch work to it?